### DISCUSSION TITLE
Mathematical Notation Input Support

### DISCUSSION THREAD

**Discussion by alonzyl** [intellibar/main#32] [Ideas]
Currently whenever a math formula appears in the conversation, it is displayed in its raw form rather than in mathematical notation. As someone who frequently engages in discussions involving mathematical concepts, this limitation makes it challenging to interpret and work with formulas.
<img width="522" alt="Screenshot 2024-03-11 at 10 55 03" src="https://github.com/intellibar/main/assets/88384677/d6750992-952a-41e9-8ac4-5ce29a92752d">

**Comment by astoilkov** [comment]
I like it! I wonder how we can detect it's a mathematical notation. Have you seen this done somewhere else or have ideas on how to detect if certain text is mathematical?
